internal/app/connectconformance/config.go

```diff
@@ -62,7 +62,7 @@ func parseConfig(configFileName string, data []byte) ([]configCase, error) {
 			Path: configFileName,
 		}
 		if err := opts.Unmarshal(data, &config); err != nil {
-			return nil, err
+			return nil, ensureFileName(err, configFileName)
```

Comment:
The protoyaml package tries to report good errors with file and line numbers. However, if it first uses a YAML library to parse the file into an AST. And if that step fails, it returns an error that has no filename in it, which is very confusing for users.